docs/source/usage.rst

```diff
@@ -438,15 +442,19 @@ This can be useful for instance to parse environment variables:
     ...         }
     ...     }
     ... )
+    >>> def show(x):
+    ...     print(f"type: {type(x).__name__}, value: {x}")
```

Comment:
We can probably add this function to the top of the testsetup or somewhere else early on.

Reply:
I moved it up and used it in more places in e090055
I didn't put it in testsetup because it doesn't appear in the doc and people may wonder exactly what is this function.